src/wallet/db.cpp

```diff
@@ -342,8 +348,55 @@ bool BerkeleyBatch::Recover(const fs::path& file_path, void *callbackDataIn, boo
         return false;
     }
 
-    std::vector<BerkeleyEnvironment::KeyValPair> salvagedData;
-    bool fSuccess = env->Salvage(newFilename, salvagedData);
+    std::vector<KeyValPair> salvagedData;
+    bool fSuccess = false; //env->Salvage(newFilename, salvagedData);
```

Comment:
nit: I'm not sure this comment is helpful

Reply:
Removed